src/web/next.config.js

```diff
@@ -13,7 +13,7 @@ const dotenv = require('dotenv');
 const loadApiUrlFromEnv = (envFile) => dotenv.config({ path: envFile });
 
 // ENV Variables we need to forward to next by prefix with NEXT_PUBLIC_*
-const envVarsToForward = ['API_URL', 'IMAGE_URL'];
+const envVarsToForward = ['API_URL', 'IMAGE_URL', 'AUTH_URL', 'POSTS_URL'];
```

Comment:
I think it's better if we do this in a different PR. We need to change more things to fix the `auth` issue.